source/point.h

```diff
@@ -1,42 +1,46 @@
 #pragma once
 #include "common.h"
+#include "constants_geom.h"
 
-ld deg_to_rad(ld d) { return d * PI / 180.0; }
-ld rad_to_deg(ld r) { return r * 180.0 / PI; }
+ld deg_to_rad(ld d) { return d * PI / 180.0;}
```

Comment:
As a general note I prefer to keep the space at the end, it makes it a bit more readable. I don't mind removing them if they take up too much space tho (did we save lines by removing these spaces?)

Reply:
there were lots of lines ending with ";}". I changed them to "; }". I agree that it looks nicer